pylint/utils/utils.py

```diff
@@ -247,36 +247,21 @@ def get_global_option(
 def get_global_option(
     checker: BaseChecker,
     option: GLOBAL_OPTION_NAMES,
-    default: T_GlobalOptionReturnTypes | None = None,
+    default: T_GlobalOptionReturnTypes | None = None,  # pylint: disable=unused-argument
 ) -> T_GlobalOptionReturnTypes | None:
```

Comment:
```suggestion
) -> T_GlobalOptionReturnTypes | None:  # pragma: no cover
```
Should we add a ``pragma: no cover`` ?

Reply:
It's covered by the deprecation tests. I added it to all the other methods because the deprecation tests only test the first few lines of those methods and all the different `ifs` within them didn't get covered. Seemed easier to just exclude them.